docs/source/usersguide/input.rst

```diff
@@ -559,12 +559,151 @@ The following code snippet illustrates the instantiation of the ``CPUSolver`` fo
+
+To illustrate the effect of this solver, a fixed source problem is chosen. This problem involves a :math:`200 \times 200` grid of water cells with a cell in the upper left filled with a fixed source in the first group. A code snippet is presented below showing the initialization of the source and solver for this problem.
+
+.. code-block:: python
+
+  # Initialize the OpenMOC solver
+  solver = openmoc.CPUSolver(track_generator)
+  solver.setNumThreads(num_threads)
+  solver.setConvergenceThreshold(tolerance)
+
+  # Set the flux in the source cell to unity and compute the flux
+  solver.setFixedSourceByCell(source_cell, 1, 1.0)
+  solver.computeFlux(max_iters)
+
+The first group flux is plotted below. All other flux plots are zero throughout the entire geometry because the scattering source is not accounted for in ``computeFlux(...)`` and neutrons are always born in the first group. By using the ``computeFlux(...)`` routine, OpenMOC is directed to only use the provided fixed source and not to update for fission or scattering. Notice that there are prominent ray effects since there is no scattering in this problem, an inherent characteristic of MOC solvers. The MOC solver is defined with 4 azimuthal angles for the figure on the left and 32 azimuthal angles for the figure on the right. As the number of angles increases, the effect is not as noticeable but is still present.
+
+.. _figure_fixed_source_flux_calc:
+
+.. table::
+
+   +--------------------------------------------------------+--------------------------------------------------------+
+   | .. _figa:                                              | .. _figb:                                              |
+   |                                                        |                                                        |
+   | .. image:: ../../img/fs-flux-calc-4-angles-group-1.png | .. image:: ../../img/fs-flux-calc-32-angles-group-1.png|
+   |   :width: 50 %                                         |   :width: 50 %                                         |
+   |   :align: right                                        |   :align: left                                         |
+   +--------------------------------------------------------+--------------------------------------------------------+
+
+While this case seems ill-suited for the ``computeFlux(...)`` routine, the ``computeFlux(...)`` is very useful for cases where the total source is known and can be defined by the user. For instance, if the total source :math:`S_g` for energy group :math:`g` is defined to be a cosine distribution such as
```

Comment:
I was simply pointing out that you are missing a "the" before the second
use of "computeFlux"
On Jan 24, 2016 9:39 PM, "Geoffrey Gunow" <notifications@github.com> wrote:

> In docs/source/usersguide/input.rst
> <https://github.com/mit-crpg/OpenMOC/pull/250#discussion_r50653526>:
>
> > +
> > +The first group flux is plotted below. All other flux plots are zero throughout the entire geometry because the scattering source is not accounted for in ``computeFlux(...)`` and neutrons are always born in the first group. By using the ``computeFlux(...)`` routine, OpenMOC is directed to only use the provided fixed source and not to update for fission or scattering. Notice that there are prominent ray effects since there is no scattering in this problem, an inherent characteristic of MOC solvers. The MOC solver is defined with 4 azimuthal angles for the figure on the left and 32 azimuthal angles for the figure on the right. As the number of angles increases, the effect is not as noticeable but is still present.
> > +
> > +.. _figure_fixed_source_flux_calc:
> > +
> > +.. table::
> > +
> > +   +--------------------------------------------------------+--------------------------------------------------------+
> > +   | .. _figa:                                              | .. _figb:                                              |
> > +   |                                                        |                                                        |
> > +   | .. image:: ../../img/fs-flux-calc-4-angles-group-1.png | .. image:: ../../img/fs-flux-calc-32-angles-group-1.png|
> > +   |   :width: 50 %                                         |   :width: 50 %                                         |
> > +   |   :align: right                                        |   :align: left                                         |
> > +   +--------------------------------------------------------+--------------------------------------------------------+
> > +
> > +While this case seems ill-suited for the ``computeFlux(...)`` routine, the ``computeFlux(...)`` is very useful for cases where the total source is known and can be defined by the user. For instance, if the total source :math:`S_g` for energy group :math:`g` is defined to be a cosine distribution such as
>
> I don't understand this comment
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-crpg/OpenMOC/pull/250/files#r50653526>.
>